lib/receipt.rb

```diff
@@ -0,0 +1,15 @@
+require 'order'
+
+module Receipt
```

Comment:
I like the way you have made a separate class for the total order that makes it all look very clear 👍